skimage/filters/tests/test_gaussian.py

```diff
@@ -62,8 +62,25 @@ def test_multichannel():
 
 
 def test_preserve_range():
+    """
+    Testing preserve_range param
+    """
+    ones = np.array([[1, 1], [1, 1]], dtype=np.int64)
+    filtered_ones = gaussian(ones, preserve_range=False)
+    assert(np.all(filtered_ones == filtered_ones[0]))
+
```

Comment:
Then I think it would make more sense to go along these lines:
```py
filtered_preserved = filters.gaussian(ones, preserve_range=True)
filtered_default = filters.gaussian(ones)
assert testing.assert_array_equal(filtered_default, filtered_preserved)
assert np.all(filtered_default == 1.)
```
What do you think?

Reply:
For now it is not necessary. 
But assuming that some changes may occur in the future - I would add that too

Reply:
I will mark this suggestion as resolved since we are not immediately changing the default to True